bokeh/io.py

```diff
@@ -436,7 +433,9 @@ def _show_file_with_state(obj, state, new, controller):
 
 def _show_jupyter_doc_with_state(obj, state, notebook_handle):
     comms_target = make_id() if notebook_handle else None
-    publish_display_data({'text/html': notebook_div(obj, comms_target)})
+    (script, div) = notebook_content(obj, comms_target)
+    publish_display_data({HTML_MIME_TYPE: div})
+    publish_display_data({JS_MIME_TYPE: script, EXEC_MIME_TYPE: ""}, metadata={EXEC_MIME_TYPE: {"id": obj._id}})
```

Comment:
publishing two messages here isn't ideal. I should refactor the ``script`` to create and embed a div.

Reply:
I think to two messages is fine. Now that I realize it's an option, I'd personally rather keep the JS scripts simpler in any way possible.

Reply:
I'm leaning towards agreeing at this point. Especially since notebook apps don't require a div, but a script that the mime renderer handles.